src/main/resources/public/raml/subscription.raml

```diff
@@ -80,4 +81,17 @@
             body:
               application/json:
                 schema: subscriptionResponse
-        
\ No newline at end of file
+
+title: Instance information
```

Comment:
Indentation looks to not be correct here. Or the indentation is wrong in the beginning of the file.